src/test/java/io/github/treesitter/jtreesitter/QueryTest.java

```diff
@@ -288,5 +288,17 @@ void findMatches() {
                         "Foo", matches.getFirst().captures().getFirst().node().getText());
             });
         }
+
+        // Verify that capture count is treated as `uint16_t` and not as signed Java `short`
+        try (var tree = parser.parse(";".repeat(Short.MAX_VALUE + 1)).orElseThrow()) {
```

Comment:
This test takes a relatively long time compared to the other ones (~3 seconds on CI); please let me know if I should remove it again.